proposals/lip-0039.md

```diff
@@ -585,6 +585,11 @@ def verify(queryKeys: list[bytes], proof: Proof, root: bytes) -> bool:
             continue
 
         # q is an inclusion proof for another leaf node.
```

Comment:
Should be here in the comment `non-inclusion` proof?
If proof is inclusion then `key == query.key` and in line 585 it's already skipped to the beginning of the `for` loop.

Reply:
It's an inclusion proof for _another_ leaf node, hence a non-inclusion proof for the queried key.